elasticsearch.md

```diff
@@ -1487,63 +1488,166 @@ ReadonlyREST can write events very similarly to Logstash into to a series of ind
+#### The 'log' output specif configurations
 
-Example: tell ROR to write on monthly index.
+The `log` output uses a dedicated logger to write the audit events in files. The logger applies the `INFO` log level to the appended audit events.
 
+To configure audit events writing to logs, you need to add to the `readonlyrest` configuration the following settings:
 ```text
 readonlyrest:
   audit:
-    collector: true
-    index_template: "'custom-prefix'-yyyy-MM"  # <--monthly pattern
+    enabled: true
+    outputs: 
+    - type: log
   ...
 ```
 
-**⚠️IMPORTANT**: notice the single quotes inside the double quoted expression. This is the same syntax used for [Java's SimpleDateFormat](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html).
+In the next step you need to configure the logger for audit events in the `$ES_PATH_CONF/config/log4j2.properties`. Here is an example config with default logger name:
+```text
+appender.readonlyrest_audit_rolling.type = RollingFile
+appender.readonlyrest_audit_rolling.name = readonlyrest_audit_rolling
+appender.readonlyrest_audit_rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}readonlyrest_audit.log
+appender.readonlyrest_audit_rolling.layout.type = PatternLayout
+appender.readonlyrest_audit_rolling.layout.pattern = [%d{ISO8601}] %m%n
+appender.readonlyrest_audit_rolling.filePattern = readonlyrest_audit-%i.log.gz
+appender.readonlyrest_audit_rolling.policies.type = Policies
+appender.readonlyrest_audit_rolling.policies.size.type = SizeBasedTriggeringPolicy
+appender.readonlyrest_audit_rolling.policies.size.size = 1GB
+appender.readonlyrest_audit_rolling.strategy.type = DefaultRolloverStrategy
+appender.readonlyrest_audit_rolling.strategy.max = 4
+
+
+logger.readonlyrest_audit.name = readonlyrest_audit   # required logger name, must be the same as the one defined in `readonlyrest.yml` 
+logger.readonlyrest_audit.appenderRef.ror_audit.ref = readonlyrest_audit_rolling    
+logger.readonlyrest_audit.additivity = false          # set to false to use only desired appenders
+```
+All settings are up to you. The only required entry is the logger name `logger.{your-logger-name}.name = {your-logger-name}`.
+The default logger name is the `readonlyrest_audit`.
+
+If you want to set a custom logger name for the `log` output, add the `logger_name` setting for the given output:
+```text
+readonlyrest:
+  audit:
+    enabled: true
+    outputs: 
+    - type: log
+      logger_name: custom-logger-name
+  ...
+```
 
-### Custom audit log serializer
+### Extended audit
 
-You can write your own custom audit log serializer class, add it to the ROR plugin class path and configure it through the YAML settings.
+If you want to log the request content then an additional serializer is provided. This will log the entire user request within the content field of the audit event. To enable, configure the `serializer` parameter as below.
```

Comment:
IMO we should link the audit event example starting in line 1459 and say that this is the default format of our audit event. And mention that it can be customized using sth we call Serializer. We should also add info how the default serializer is called (it can be extended by our users). Then we should about other predefined serializers.